keel-core/src/test/kotlin/com/netflix/spinnaker/keel/diff/ResourceDiffTests.kt

```diff
@@ -47,13 +54,13 @@ internal class DiffTests : JUnit5Minutests {
       Fixture()
     }
 
-    context("same parent two children") {
-//      test("diffing works") {
-//        val obj1: Parent = Child1("common", "myprop")
-//        val obj2: Parent = Child2("common", "anotherProp")
-//        val resourceDiff = ResourceDiff(obj1, obj2)
-//        expectThat(resourceDiff.diff.hasChanges()).isEqualTo(true)
-//      }
+    SKIP - context("delta is the type of a polymorphic property") {
```

Comment:
Test skipped until #317 is fixed

Reply:
woah I didn't know that this was a thing you could do! cool.

Also, shouldn't this PR fix #317 ?